.travis.yml

```diff
@@ -18,74 +16,57 @@ env:
         - CONFIG=Release
         - CONFIG=Debug
 
+addons:
+    apt:
+        sources:
+            # Add the modern toolchain repositories (llvm repo depends on ubunut-toolchain-r + gcc)
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-precise-3.8
+            # Add the MongoDB 3.2 repository
+            - mongodb-3.2-precise
+        packages:
+            - mongodb-org-server
+            - g++-6
+            - clang-3.8
+
 # Setup required repositories before installation
 before_install:
-    # Add the modern toolchain repositories (llvm repo depends on ubunut-toolchain-r + gcc)
-    - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-    - sudo add-apt-repository -y 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.8 main'
-    - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
-
     # Get the latest CMake
     - curl -O https://cmake.org/files/v3.2/${CMAKE_VERSION}.tar.gz
```

Comment:
It doesn't work because the repository's version of cmake requires a different version of ubuntu than the one on the container, I'll just leave as is